### ISSUE TITLE
auto-save feature (is awesome but) moves cursor to end of text box

### ISSUE THREAD

**Issue by vrooje** [zooniverse/Panoptes-Front-End#772]
Am editing my science-case page. Really liking the auto-save and the green highlight of the title above the text box works pretty well (though a small "Saved" that fades in next to the title as the green fades, then stays until a change is made, would be reassuring).

However, every time the auto-save completes the cursor moves to the end of the text box and I have to re-locate where I was before it engaged. Can that be fixed?

**Comment by vrooje** [comment]
Yeah, I've just been dealing with this in the workflow page too and it's much more difficult when you're making changes and move to another box and then the previous box tries to save. In one case I lost text because the saving of box A caused a refresh/revert of the text in box B.

**Comment by vrooje** [comment]
This is hugely improved, although I may have just broken it:

I changed some help text, and moved my cursor to the Main Text box to make the Help Text box save, which it did. I went to the classification interface and refreshed, then clicked through to pop up the new help text, which displayed my changes.

When I went back to the project builder, the "Main Text" title text was highlighted in red (I'd left my cursor in there but hadn't made changes) and some of the changes I'd previously made, saved and confirmed were present in the Help Text were missing from that box. I re-updated the text, saved, and all was fine.

It's happened twice so far.
